week06_policy_based/a2c-optional.ipynb

```diff
@@ -147,7 +147,7 @@
     "The formula for the value targets is simple:\n",
     "\n",
     "$$\n",
-    "\\hat v(s_t) = \\sum_{t'=0}^{T - 1}\\gamma^{t'}r_{t+t'} + \\gamma^T \\hat{v}(s_{t+T}),\n",
+    "\\hat v(s_t) = \\left( \\sum_{t'=0}^{T - 1 - t'} \\gamma^{t'}r_{t+t'} \\right) + \\gamma^T \\hat{v}(s_{t+T}),\n",
```

Comment:
This cannot be correct. You define `t'` as the counter for `\sum`, and then use it in the upper limit of the same sum.

Reply:
Fix old typo = make new typo.
Correct variant in upper bount is T-1-t (t without apostrophe)

sum_{t'=0}^{T - 1 - t}

Reply:
Thanks!